en/api-gateway/concepts/extensions/object-storage.md

```diff
@@ -8,11 +8,11 @@ The `x-yc-apigateway-integration:object_storage` extension passes the request ha
 
 Parameter | Type | Description
 ---- | ---- | ----
-`bucket` | `string` | Name of the [bucket](../../../storage/concepts/bucket.md) name.
+`bucket` | `string` | Name of the [bucket](../../../storage/concepts/bucket.md).
 `object` | `string` | [Object](../../../storage/concepts/object.md) name. Supports parameter standardization from the path of the original request. <br>Parameters are substituted in `object`.
 `error_object` | `string` | Optional parameter. [Object](../../../storage/concepts/object.md) name returned if HTTP error code 4xx is received instead of `object`.
 `presigned_redirect` | `bool` | If the value is `true`, a [pre-signed URL](../../../storage/concepts/pre-signed-urls.md) is generated and a redirect is returned to the client.
-`service_account_id` | `string` | ID of the service account used for authorization when accessing {{ objstorage-short-name }}. If the parameter is omitted, the value of the [top-level](./index.md#top-level) `service_account_id` parameter is used. If there is no top-level parameter, the function is invoked without authorization.
+`service_account_id` | `string` | ID of the service account used for authorization when accessing {{ objstorage-short-name }}. If the parameter is omitted, the value of the [top-level](./index.md#top-level) `service_account_id` parameter is used.
```

Comment:
If there is no top-level parameter, the object is accessed without authorization.